zerver/webhooks/beeminder/fixtures/derail.json

```diff
@@ -0,0 +1,137 @@
+{
+    "goal": {
+        "slug": "gainweight",
+        "title": "Have to increase weight to get rid of this skinny bones !",
```

Comment:
I appreciate that you're avoiding using a weightloss goal. :) How about this as a title?

> "Have to get back to the shape I want!"